### PR TITLE
feat: New Button theme: text

### PR DESCRIPTION
**Pull request by niklabh**
Screenshot from 2022-04-14 12-02-51

Screenshot from 2022-04-14 12-03-03

Screenshot from 2022-04-14 12-03-21

### CODE REVIEW

File: src/components/Button/styles/standardThemes.ts [hunk 1]
```diff
@@ -155,3 +155,32 @@ export const translucent = css`
         fill: ${color.white};
     }
 `;
+
+export const text = css`
+    background-color: ${color.white};
```
Comment (Y0moo) [inline src/components/Button/styles/standardThemes.ts:160]: Let's replace it to `transparent` here

File: src/components/Button/styles/standardThemes.ts [hunk 2]
```diff
@@ -155,3 +155,32 @@ export const translucent = css`
...
+        color: ${color.paleCerulean};
+        background-color: ${color.blueLight};
+        border-color: transparent;
+
+        svg {
+            fill: ${color.paleCerulean};
+        }
+    }
+
+    :active {
```
Comment (Y0moo) [inline src/components/Button/styles/standardThemes.ts:175]: Following the style in Figma, we need to have a "gray" bg in active state

image

**Review by Y0moo — CHANGES_REQUESTED** [review]

File: src/components/Button/styles/standardThemes.ts [hunk 3]
```diff
@@ -155,3 +155,32 @@ export const translucent = css`
...
+
+export const text = css`
+    background-color: ${color.white};
+    border-color: transparent;
+
+    color: ${color.blue};
+
+    :hover {
+        color: ${color.paleCerulean};
+        background-color: ${color.blueLight};
```
Comment (Y0moo) [inline src/components/Button/styles/standardThemes.ts:167]: Should be there `background-color: ${color.blueSkyDark};`

**Comment by Y0moo** [comment]
Great work @niklabh, please check my comments

**Comment by niklabh** [comment]
Done

**Review by BillyG83 — CHANGES_REQUESTED** [review]
nit sure this works, it looks like `secondary` when i use `text`

what is the required result? Just text but acts like a button? Maybe it would need less padding also to sit inline with text?

**Comment by BillyG83** [comment]
Screenshot 2022-04-18 at 13 52 12

from my testing

**Comment by Y0moo** [comment]
weird, why do you have themes as a string input and not as a group of radios in the canvas? It should be like: 

image

**Comment by niklabh** [comment]
> weird, why do you have themes as a string input and not as a group of radios in the canvas? It should be like:
> 
> image

Strange. No idea why its happening

**Comment by BillyG83** [comment]
> why do you have themes as a string input and not as a group of radios in the canvas? It should be like:

i think once you have too many options Storybook shows the dropdown instead of radios to save space

**Pull request closed without merging**